src/controllers/shopController.ts

```diff
@@ -51,11 +45,9 @@ export const getShop = async (req: Request, res: Response) => {};
 
 export const getAllShops = async (req: Request, res: Response) => {
```

Comment:
For methods that are just used in testing, we should document or comment it for clarity